### PR TITLE
Added GraalVM to allow native compilation on node for binary builds

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

**Review by Sgitario — APPROVED** [review]

**Review by Sgitario — APPROVED** [review]

File: kogito-cloud-operator.jenkinsfile [hunk 1]
```diff
@@ -286,6 +288,25 @@ void installOpenshiftClient(){
     """
 }
 
+void installGraalVMWithNative(){
+    echo "installGraalVMWithNative"
+    sh """       
+        rm -rf ${WORKSPACE}/graalvm*
+        RELEASE_VERSION=19.3.1
+        curl -LO https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-\${RELEASE_VERSION}/graalvm-ce-java11-linux-amd64-\${RELEASE_VERSION}.tar.gz
+        echo "hello"
```
Comment (sutaakar) [inline kogito-cloud-operator.jenkinsfile]: debugging leftover?

File: kogito-cloud-operator.jenkinsfile [hunk 2]
```diff
@@ -286,6 +288,25 @@ void installOpenshiftClient(){
     """
 }
 
+void installGraalVMWithNative(){
+    echo "installGraalVMWithNative"
+    sh """       
+        rm -rf ${WORKSPACE}/graalvm*
+        RELEASE_VERSION=19.3.1
```
Comment (sutaakar) [inline kogito-cloud-operator.jenkinsfile]: Wouldn't it have more sense to put to `environment` above so we don't have to go through whole script when we need to raise the version?

File: kogito-cloud-operator.jenkinsfile (hunk 1 above)
Comment (radtriste) [reply kogito-cloud-operator.jenkinsfile]: removed

File: kogito-cloud-operator.jenkinsfile (hunk 2 above)
Comment (radtriste) [reply kogito-cloud-operator.jenkinsfile]: right
will set it

Comment (radtriste) [reply kogito-cloud-operator.jenkinsfile]: done

**Review by sutaakar — APPROVED** [review]

**Pull request merged**